resolwe_bio/processes/workflows/rnaseq.yml

```diff
@@ -229,3 +229,103 @@
           mode: union
           stranded: '{{input.stranded}}'
           id_attribute: '{{input.id_attribute}}'
+
+- slug: workflow-meta-rnaseq
+  name: RNA-Seq workflow (meta)
+  requirements:
+    expression-engine: jinja
+  data_name: "Pipeline ({{ reads|sample_name|default('?') }})"
+  version: 0.0.1
+  type: data:workflow:rnaseq:htseq
+  category: workflow:primary
+  persistence: RAW
+  description: >
+    Run RNA-Seq workflow
+  input:
+    - name: reads
+      label: Input reads
+      type: data:reads:fastq
+    - name: genome
```

Comment:
So a simple input instead of this one would be:
```
    - name: genome_and_annotation
       label: Organism
       type: basic:string
       default: hs
       choices:
         - label: Homo sapiens
           value: hs
         - label: Mus musculus
           value: mm
```
Then you should fetch the correct genome in the process itself (using resdk). The logic which genome to use should be part of the process. We want to remove the decision logic from the configuration.